scripts/run_etcd.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
-
+git clone https://go.googlesource.com/go
+cd go
+git checkout master
+cd src
+./all.bash
+GOROOT_BOOTSTRAP=$GOROOT
```

Comment:
@AdoHe Your version seems better, you can overwrite my version latter, as I want to pass the test. =,=